frontend/src/app/visualization/visualization.component.html

```diff
@@ -105,8 +105,8 @@
                         (error)="setErrorMessage($event)"
                         (totalTokenCountAvailable)="showTokenCountOption = $event"></ia-histogram>
                 </ng-container>
-                <ia-wordcloud *ngSwitchCase=" 'wordcloud' " [corpus]="corpus" [queryModel]="queryModel" [visualizedField]="visualizedField"
-                    [resultsCount]="resultsCount" [asTable]="freqtable" [palette]="palette"
+                <ia-wordcloud *ngSwitchCase=" 'wordcloud' " [queryModel]="queryModel"
```

Comment:
What's with the whitespace around `'wordcloud'`? Is that interpreted? Does it break if we remove it?

Reply:
Knowing I-analyzer, this is either strange formatting that was just never changed, or removing the space will somehow break all CSV downloads.

Probably the former.